openshift_cli_installer/cli.py

```diff
@@ -275,11 +277,14 @@ def main(**kwargs):
     ocm_managed_clusters = aws_managed_clusters + gcp_osd_clusters
 
     check_ocm_managed_existing_clusters(clusters=ocm_managed_clusters, create=create)
-    is_region_support_hypershift(hypershift_clusters=hypershift_clusters)
-    is_region_support_aws(clusters=aws_ipi_clusters + aws_managed_clusters)
+    is_region_support_hypershift(hypershift_clusters=hypershift_clusters, create=create)
```

Comment:
Why not using `if create:` before all `is_region_support_[hypershift/aws/gcp]`
instead of within each function?
```suggestion
if create:
    is_region_support_hypershift(hypershift_clusters=hypershift_clusters, create=create)
```

Every function starts with an `if create:` without an else referral or any other reference to the action itself.

Reply:
Sorry I couldn't indent properly within the suggestion^

Reply:
to keep the function as "clean" as possible and reduce complexity  (i.e - logic in the called functions)

Reply:
but will change